provisioning/src/main/scala/io/phdata/provisioning/WorkspaceRequestProvisioning.scala

```diff
@@ -93,7 +93,7 @@ trait WorkspaceRequestProvisioning {
             ),
             kafkaTopicsNotEnabledMessage
           )
-          _ <- ImpalaService.invalidateMetadata(workspace.id.get)(workspaceContext.context)
+          _ <- ImpalaServiceImpl.invalidateMetadata(workspace.id.get)(workspaceContext.context)
```

Comment:
It might have been easier to create a class out of the ImpalaServiceImpl (but the object is fine, it's working)

Reply:
Yes we could transform it to a class but I think for now have as an object is fine.